sdk/script/acquisition-sdk.ts

```diff
@@ -104,8 +104,8 @@ export class AcquisitionManager {
 
             if (response.statusCode !== 200) {
                 let errorMessage;
-                if (response.statusCode === 0){
-                    errorMessage = `Couldn't send request to ${requestUrl}. One of the possible reasons for that might be connection problems. Please, check your internet connection.`;
+                if (response.statusCode === 0) {
+                    errorMessage = `Couldn't send request to ${requestUrl}, xhr.statusCode = 0 was returned. One of the possible reasons for that might be connection problems. Please, check your internet connection. Also, see https://stackoverflow.com/a/26451773 for more details`;
```

Comment:
Don't sure if it's a good idea to provide 3rd party URLs in our logs, would be better to provide a link to CodePush issue/comment on github if we really need it.

Reply:
Got it, I removed it.